box-content-sdk/src/test/java/com/box/androidsdk/content/requests/BoxUploadRequestTest.java

```diff
@@ -0,0 +1,116 @@
+package com.box.androidsdk.content.requests;
+
+import android.content.Context;
+
+import com.box.androidsdk.content.BoxApiFile;
+import com.box.androidsdk.content.auth.BoxAuthentication;
+import com.box.androidsdk.content.listeners.ProgressListener;
+import com.box.androidsdk.content.models.BoxSession;
+import com.box.androidsdk.content.testUtil.PowerMock;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+import org.mockito.Mock;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Date;
+
+/**
+ * Tests for upload requests
+ */
+
+@PrepareForTest({ BoxHttpResponse.class,  BoxRequest.class, BoxRequestMultipart.class})
```

Comment:
Can we use variable instead of values? Eg.

```java
String expectedFilename = "file":
File file = new File(expectedFilename);
...
Assert.assertEquals(expectedFilename, uploadRequest.getFileName());
```
```java
String expectedSha1 = "sha1"
uploadRequest.setSha1(expectedSha1);
Assert.assertEquals(expectedSha1, uploadRequest.getSha1());